openc3/spec/models/target_model_spec.rb

```diff
@@ -26,10 +26,16 @@
 require 'openc3/models/microservice_model'
 require 'openc3/utilities/aws_bucket'
 
+module Aws
+  autoload(:S3, 'openc3/utilities/s3_autoload.rb')
```

Comment:
Autoload is an optimization for large libraries.  It should only be used if you don't know if a symbol will be accessed or not by the user.   In a spec, file just use regular require.

Reply:
Corrected